### PR TITLE
Issue #SB-28199 fix: retaining activity dashboard option on refresh

### PR DESCRIPTION
**Pull request by balakrishna10**

### CODE REVIEW

File: src/app/client/src/app/modules/groups/components/activity/activity-list/activity-list.component.ts [hunk 1]
```diff
@@ -97,7 +98,7 @@ export class ActivityListComponent implements OnInit, OnDestroy {
       this.addTelemetry('activity-card', [{id: _.get(event, 'data.identifier'), type: _.get(event, 'data.resourceType')}]);
       const options = { relativeTo: this.activateRoute, queryParams: { primaryCategory: _.get(event, 'data.primaryCategory'),
       title: activityType, mimeType: _.get(event, 'data.mimeType'), groupId: _.get(this.groupData, 'id')}};
-      this.activityDashboardService.isActivityAdded = true; // setting this value to enable or disable the activity dashboard button in activity-dashboard directive
+      sessionStorage.setItem(sessionKeys.ISACTIVITYADDED, 'true') // setting this value to enable or disable the activity dashboard button in activity-dashboard directive
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/groups/components/activity/activity-list/activity-list.component.ts:101]: Dont use sessions for more info.

**Review by vinukumar-vs — APPROVED** [review]

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]
Lets check what information are we dumping in session.

File: src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ import { IImpressionEventInput } from '@sunbird/telemetry';
 import { CsGroupAddableBloc } from '@project-sunbird/client-services/blocs';
 import { VIEW_ACTIVITY, CATEGORY_SEARCH } from '../../../interfaces/telemetryConstants';
 import { ActivityDashboardService } from '@sunbird/shared';
-// import { sessionKeys } from '../../..';
+import { sessionKeys } from '../../..';
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts:24]: please make the correct import

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts (hunk 2 above)
Comment (balakrishna10) [reply src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts]: updated now.

**Review by rajeevsathish — APPROVED** [review]

**Pull request merged**